packages/beacon-node/test/unit/chain/validation/syncCommittee.test.ts

```diff
@@ -159,24 +147,19 @@ describe("Sync Committee Signature validation", function () {
     const syncCommittee = getSyncCommitteeSignature(currentSlot, validatorIndexInSyncCommittee);
     const headState = generateCachedAltairState({slot: currentSlot}, altairForkEpoch);
 
-    chain.getHeadState.returns(headState);
-    chain.bls = new BlsVerifierMock(true);
+    chain.getHeadState.mockReturnValue(headState);
 
     const subnet = 3;
     const {slot, validatorIndex} = syncCommittee;
     const prevRoot = "0x1234";
     chain.seenSyncCommitteeMessages.add(slot, subnet, validatorIndex, prevRoot);
-    expect(chain.seenSyncCommitteeMessages.get(slot, subnet, validatorIndex)).to.be.equal(
-      prevRoot,
-      "cache should return prevRoot"
-    );
+    expect(chain.seenSyncCommitteeMessages.get(slot, subnet, validatorIndex)).toBe(prevRoot);
     // but forkchoice head is message root
-    forkchoiceStub.getHeadRoot.returns(toHexString(syncCommittee.beaconBlockRoot));
+    forkchoiceStub.getHeadRoot.mockReturnValue(toHexString(syncCommittee.beaconBlockRoot));
     await validateGossipSyncCommittee(chain, syncCommittee, subnet);
     // should accept the message and overwrite prevRoot
-    expect(chain.seenSyncCommitteeMessages.get(slot, subnet, validatorIndex)).to.be.equal(
-      toHexString(syncCommittee.beaconBlockRoot),
-      "should add message root to seenSyncCommitteeMessages"
+    expect(chain.seenSyncCommitteeMessages.get(slot, subnet, validatorIndex)).toBe(
+      toHexString(syncCommittee.beaconBlockRoot)
```

Comment:
Aren't we losing information here if the assertion fails?

> "should add message root to seenSyncCommitteeMessages"

This message seems helpful to me instead of just seeing value x does not equal value y

Reply:
Yes we are loosing some information here, that will require to some effort to get the tests in right order to get that information.